src/migrations/20200102145246-create-location.js

```diff
@@ -5,6 +5,9 @@ export const up = (queryInterface, Sequelize) => queryInterface.createTable('Loc
     primaryKey: true,
     type: Sequelize.INTEGER
   },
+  code: {
```

Comment:
@gadishimwe instead of editing an existing migration, you should create a new one that contains the changes you want.